src/terminal/terminal_handlers.c

```diff
@@ -779,13 +919,30 @@ int guac_terminal_csi(guac_terminal* term, unsigned char c) {
 
                     /* Foreground */
                     else if (value >= 30 && value <= 37)
```

Comment:
If you're referring to 30 and 37 here, these are the values from the standard defining the console codes used by the Guacamole terminal emulator.

Might not be a bad idea to do go through the terminal emulator code and add constants for all of these, but I'd say we should do that as a separate cleanup issue, rather than having these be the only two such constants.